utils/mirror_vfd/mirror_remote.c

```diff
@@ -78,7 +78,7 @@ mirror_log(struct mirror_log_info *info, unsigned int level, const char *format,
  * ----------------------------------------------------------------------------
  */
 void
-mirror_log_bytes(struct mirror_log_info *info, unsigned int level, size_t n_bytes, const unsigned char *buf)
+mirror_log_bytes(struct mirror_log_info *info, unsigned int level, ssize_t n_bytes, const unsigned char *buf)
```

Comment:
ssize_t is not in the C standard (it's POSIX). We should avoid using it in new code.

Reply:
reference please?

Reply:
@lrknox , feel free to close this PR and apply your patch that worked for you.